specification/communication/data-plane/role/preview/2022-01-01-preview/CommunicationRole.json

```diff
@@ -0,0 +1,439 @@
+            }
+          },
+          "200": {
+            "description": "OK - Returns the communication role.",
+            "schema": {
+              "$ref": "#/definitions/CommunicationRoleResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get a Communication Role": {
+            "$ref": "./examples/GetRole.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "CommunicationRoleCreatePermissions": {
+      "description": "The object containing the various types of permissions.",
+      "required": [
+        "dataActions"
+      ],
+      "type": "object",
+      "properties": {
+        "dataActions": {
+          "description": "The list of data oriented action permissions.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CommunicationRolePermissionActions"
+          }
+        }
+      }
+    },
+    "CommunicationRoleCreateRequest": {
+      "required": [
+        "displayName",
+        "isEnabled",
+        "permissions"
+      ],
+      "type": "object",
+      "properties": {
+        "displayName": {
+          "description": "The human readable name of the Role.",
+          "type": "string"
+        },
+        "permissions": {
+          "$ref": "#/definitions/CommunicationRoleCreatePermissions"
+        },
+        "isEnabled": {
+          "description": "Is Role enabled for assignments.",
+          "type": "boolean"
+        }
+      }
+    },
+    "CommunicationRoleGetAllResponse": {
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The query response content",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CommunicationRoleResponse"
+          }
+        },
+        "maxItems": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "@nextLink": {
+          "type": "string"
+        }
+      }
+    },
+    "CommunicationRolePermissionActions": {
+      "description": "The available action permissions list.",
+      "enum": [
+        "communication.services/calling/audio/receive/action",
+        "communication.services/calling/video/receive/action",
+        "muteThemselves",
+        "communication.services/calling/mute/others/audio/action",
+        "communication.services/calling/mute/others/video/action",
+        "communication.services/calling/audio/send/action",
+        "communication.services/calling/video/send/action",
+        "communication.services/calling/screenshare/send/action",
+        "communication.services/calling/mute/others/screenshare/action",
+        "communication.services/calling/recording/action",
+        "communication.services/calling/transcript/action",
+        "communication.services/calling/livecaptions/action",
+        "communication.services/calling/admitLobby/action",
```

Comment:
The above three features don't exist in ACS yet, do we still want to include permissions for them in the first RBAC preview?

Reply:
This list is not final and will have to go through spec validation first.